keyboards/splitty/config.h

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 Nicolas Schodet
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x1d50
```

Comment:
This vender ID looks to be in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x1D50

Reply:
I requested this VENDOR_ID/PRODUCT_ID to OpenMoko.

See https://github.com/openmoko/openmoko-usb-oui/blob/master/usb_product_ids.psv#L107

(Yes, APBTeam is me)

I reuse the PRODUCT_ID used for my dev boards… OK perhaps not the best idea, I will request a new one.

Reply:
Ah, that's okay then.

Reply:
Is it okay to push it like this and change the PRODUCT_ID later?